### DISCUSSION TITLE
power(base,exponential)

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17197] [How to] [answered]
Hello,

In vtiger there is the option to clear/leave empty a date field when a record, for example an issue is closed using the power(base,exponential) function with a Workflows.
Is there this option in YF? How do I manage to clear/leave a date field empty? For example, in a sales invoice I have the Expiration Date field, I would like that when the invoice is already in paid status, the Expiration Date field is cleared/leave empty.

**Comment by apcloic** [comment, marked as answer]
@krseguros In the update field task of your workflow -> Set value -> Expression -> Null
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/2930560/9628a7cf-b2a9-484d-8e62-211ee2176955)

**Reply by krseguros** [reply]
Thank you!!! Resolved.
